engine/glfw/js/library_glfw.js

```diff
@@ -588,8 +590,8 @@ var LibraryGLFW = {
     GLFW.addEventListener("mousemove", GLFW.onMousemove, true);
     GLFW.addEventListener("mousedown", GLFW.onMouseButtonDown, true);
     GLFW.addEventListener("mouseup", GLFW.onMouseButtonUp, true);
-    GLFW.addEventListener('DOMMouseScroll', GLFW.onMouseWheel, true);
-    GLFW.addEventListener('mousewheel', GLFW.onMouseWheel, true);
+    GLFW.addEventListener('DOMMouseScroll', GLFW.onMouseWheel, { capture: true, passive: false });
```

Comment:
Setting capture to true is the same as passing true to the third argument (done for the rest of the event listeners)

If passive is true it means that the function called by the even listener must not call preventDefault (will generate a warning otherwise). This is the opposite of what we want, so we set passive to false.

Reply:
According to docs wheel, mousehweel, touchstart and touchmove default to passive = true. I did not notice any warning from a call to preventDefault() for touchstart though.